tests/cli/conftest.py

```diff
@@ -1,9 +1,11 @@
+from typing import Callable, Any, Tuple
 import pytest
 import os
+from _pytest.pytester import Testdir, RunResult
 
 
 @pytest.fixture
-def run(testdir):
+def run(testdir: Testdir) -> Callable[[Tuple[Any]], RunResult]:
```

Comment:
Is there any particular reason we're not using `Callable[..., RunResult]` instead of `Callable[[Tuple[Any]], RunResult]`?

Reply:
Nope, gonna fix it